docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md

```diff
@@ -0,0 +1,333 @@
+```
+Create the PV and PVC for the database:
+
+```bash
+$ mkdir /scratch/DockerVolume/DB
+$ chmod 777 /scratch/DockerVolume/DB
+```
+Update the appropriate host/storage path in `db-pv.yaml`.
+
+Create the PV/PVC using these commands:
+
+```bash
+$ cd  weblogic-kubernetes-operator/kubernetes/samples/scripts/create-database-pv-pvc
+$ kubectl create -f db-pv.yaml
+$ kubectl create -f db-pvc.yaml
+```
+Bring up the database and database service using the following commands:
+
+```bash
+$ cd soa-kubernetes-operator/kubernetes/samples/scripts/create-soa-domain
+$ kubectl create -f db-with-pv.yaml
+```
+Verify that the database service status is Ready.
+
+```bash
+$ kubectl get pods,svc -n soans |grep soadb
+po/soadb-0   1/1       Running   0          6m
+
+svc/soadb   ClusterIP   None         <none>        1521/TCP,5500/TCP   7m
+$
+```
+
+#### Running the Repository Creation Utility to set up your database schema
+
+If you want to run RCU from a pod inside the Kubernetes cluster, you can use the Docker
+image that you built earlier as a "service" pod to run RCU.  To do this, start up a
+pod using that image as follows:
+
+```bash
+kubectl run rcu --generator=run-pod/v1 --image oracle/soa:12.2.1.3 -n soans  -- sleep infinity
+```
+
+This will create a Kubernetes deployment called `rcu` containing a pod running a container
+created from the `oracle/soa:12.2.1.3` image which will just run
+`sleep infinity`, which essentially creates a pod that we can "exec" into and use to run whatever
+commands we need to run.
+
+To get inside this container and run commands, use this command:
+
+```bash
+kubectl exec -n soans -ti rcu /bin/bash
+```
+
+When you are finished with this pod, you can remove it with this command:
+
+```bash
+kubectl delete pod rcu
+```
+
+{{% notice note %}}
+You can use the same approach to get a temporary pod to run other utilities
+like WLST.
+{{% /notice %}}
+
+##### Creating schemas
+
+Inside this pod, you can use the following command to run RCU in command-line (no GUI) mode
+to create your schemas required for SOA domains.  You will need to provide the right prefix and connect string.
+You will be prompted to enter the password for the `sys` user, and then the password to use
+for the regular schema users:
+
+```bash
+$export CONNECTION_STRING=soadb:1521/soapdb.us.oracle.com
+$export RCUPREFIX=SOA1
+
+$echo -e Oradoc_db1"\n"Welcome1 > /tmp/pwd.txt
+
+$/u01/oracle/oracle_common/bin/rcu \
+-silent \
+-createRepository \
+-databaseType ORACLE \
+-connectString $CONNECTION_STRING \
+-dbUser sys \
+-dbRole sysdba   \
+-useSamePasswordForAllSchemaUsers true \
+-selectDependentsForComponents true \
+-variables SOA_PROFILE_TYPE=SMALL,HEALTHCARE_INTEGRATION=NO \
+-schemaPrefix $RCUPREFIX  \
+-component MDS \
+-component IAU \
+-component IAU_APPEND \
+-component IAU_VIEWER \
+-component OPSS \
+-component WLS \
+-component STB \
+-component SOAINFRA \
```

Comment:
Please change this line
-component SOAINFRA \
to 
-component SOAINFRA  < /tmp/pwd.txt